src/monster.cpp

```diff
@@ -1410,32 +1410,34 @@ float monster::stability_roll() const
     return stability;
 }
 
-float monster::get_dodge() const
+float monster::get_dodge( bool include_size ) const
 {
     float ret = 0.0f;
-    switch( type->size ) {
-        case MS_TINY:
-            ret += 6;
-            break;
-        case MS_SMALL:
-            ret += 3;
-            break;
-        case MS_LARGE:
-            ret -= 2;
-            break;
-        case MS_HUGE:
-            ret -= 4;
-            break;
-        case MS_MEDIUM:
-            break; // keep default
+    if( include_size ) {
```

Comment:
Instead of sometimes handling this here and sometimes handling it elsewhere, can we just pull this out of here entirely?  It looks like it's only called about 10 times, it would be less change to apply this adjustment in each other location manually rather than dding the parameter to this mentod, and everything would be clearer.